tests/parsers/perkin_elmer_envision/elmer_envision_parser_test.py

```diff
@@ -37,3 +38,25 @@ def test_get_model() -> None:
         model.measurement_aggregate_document.measurement_identifier = ""
 
     assert model == get_model()
+
+
+@pytest.mark.parametrize("output_file", output_files)
+def test_parse_missing_file(output_file: str) -> None:
+    test_filepath = f"tests/parsers/perkin_elmer_envision/testdata/{output_file}.tsv"
+    with pytest.raises(FileNotFoundError):
+        from_file(test_filepath, VENDOR_TYPE)
+
+
+@pytest.mark.parametrize("output_file", output_files)
```

Comment:
Same here, no need to parameterize.

Reply:
Resolved in [5833c79](https://github.com/Benchling-Open-Source/allotropy/pull/72/commits/5833c79bdfb6cabff12553304bb83d5ba3dfdf67)